ui/raidboss/data/07-dt/dungeon/strayborough-deadwalk.ts

```diff
@@ -0,0 +1,257 @@
+import Conditions from '../../../../../resources/conditions';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: Add basically anything to help with the doll charges.
+// TODO: Determine safe spots for Tea Awhirl
+// TODO: Determine safe spots for Toiling Teapots
+
+export interface Data extends RaidbossData {
+  playerIsGhost: boolean;
+  solidBitterLinesNext: boolean;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'The Strayborough Deadwalk',
+  zoneId: ZoneId.TheStrayboroughDeadwalk,
+  timelineFile: 'strayborough-deadwalk.txt',
+  initData: () => {
+    return {
+      playerIsGhost: false,
+      solidBitterLinesNext: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'Strayborough Deadwalk Leonogg Malicious Mist',
+      type: 'StartsUsing',
+      netRegex: { id: '8EB1', source: 'His Royal Headness Leonogg I', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Strayborough Deadwalk Falling Nightmare',
+      type: 'StartsUsing',
+      netRegex: { id: '8EAE', source: 'His Royal Headness Leonogg I', capture: false },
+      infoText: (_data, _matches, output) => output.nightmare!(),
```

Comment:
```suggestion
      netRegex: { id: '8EAE', source: 'His Royal Headness Leonogg I', capture: false },
      delaySeconds: 3,
      infoText: (_data, _matches, output) => output.nightmare!(),
```


(Or alternatively trigger on the Ability line?)  This just felt a little early, as the reminder was gone before the adds showed up.